tests/ttnn/python_api_testing/non_working_unit_tests/grayskull/test_argmax_padding.py

```diff
@@ -0,0 +1,61 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import pytest
+import tt_lib
+from loguru import logger
+from tests.tt_eager.python_api_testing.sweep_tests import comparison_funcs
+
+
+@pytest.mark.parametrize(
+    "input_shapes",
+    (
+        (torch.Size([1, 1, 1, 10])),
+        (torch.Size([1, 1, 1, 20])),
+        (torch.Size([1, 1, 1, 3])),
+    ),
+)
+@pytest.mark.parametrize(
+    "dim",
+    (3,),
+)
+@pytest.mark.parametrize("all", (True,))
```

Comment:
```suggestion
@pytest.mark.parametrize("all", (True, False))
```
I'm not sure what the meaning of the "all" parameter is, but it seems like testing with False should also be included for proper testing.

Reply:
All is used to perform argmax without considering the dim param it is equivalent to using torch.argmax without passing dim param